docs/src/pages/[platform]/components/authenticator/customization/customization.styling.web.mdx

```diff
@@ -6,7 +6,7 @@ import { Fragment } from '@/components/Fragment';
 
 ### Styling
 
-You can customize the Authenticator's default style by using [CSS variables](/components/authenticator#amplify-css-variables).
+You can customize the Authenticator's default style by using [CSS variables](../../theming/css-variables).
```

Comment:
A lot of the docs were using an import from `/components` . This is wrong in a lot of places since `/components` assumes the URL structure is `localhost/components/authenticator` instead of `localhost/react/components/authenticator`. For React documents this would redirect correctly, however for Vue, Angular and Flutter it would redirect to the react docs, instead of the correct framework.